base_rest/restapi.py

```diff
@@ -5,15 +5,13 @@
 
 from cerberus import Validator
 
-from odoo import _
+from odoo import _, http
 from odoo.exceptions import UserError
 
 from .tools import cerberus_to_json
 
 
-def method(
-    routes, input_param=None, output_param=None, auth=None, cors=None, csrf=False
-):
+def method(routes, input_param=None, output_param=None, **kw):
```

Comment:
explicit auth, cors, csrf made them easily discoverable.

Reply:
It's on purpose... It's the only I found to know if the parameter has been explicitly provided or not. This information is required to know if the default parameters provided by the controller must be applied or not on a route.

Reply:
ah, ok, thanks